### PR TITLE
Issue/8199

### PR DESCRIPTION
**Pull request by ilci**

### CODE REVIEW

File: web/themes/custom/subtheme_videogame/src/components/views/_view--catalog.scss [hunk 1]
```diff
@@ -14,6 +14,10 @@
     margin-bottom: 2rem;
   }
 
+  &__catimage {
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/components/views/_view--catalog.scss]: If you want to create a variant you should foloww the BEM standards, e.g. `.catalog__image--somevariant`.

File: config/sync/views.view.uploaded_products.yml [hunk 2]
```diff
@@ -311,11 +363,17 @@ display:
...
-          empty: false
+          empty: true
           tokenize: false
           content:
             value: ''
             format: full_html
+          roles_fieldset:
+            roles:
+              administrator: administrator
+              product_uploader: product_uploader
```
Comment (kittiradovics) [inline config/sync/views.view.uploaded_products.yml:374]: Also please make sure that users that should not see the button, can't even access the product creation page with its URL.

**Review by kittiradovics — CHANGES_REQUESTED** [review]

File: web/themes/custom/subtheme_videogame/src/components/views/_view--catalog.scss (hunk 1 above)
Comment (ilci) [reply web/themes/custom/subtheme_videogame/src/components/views/_view--catalog.scss]: I renamed it to .catalog__image--marginless.

File: config/sync/views.view.uploaded_products.yml (hunk 2 above)
Comment (ilci) [reply config/sync/views.view.uploaded_products.yml:374]: I think it works fine: if a normal user tries to access the product creation page, (s)he gets the "Access denied You are not authorized to access this page." message.

**Review by kittiradovics — APPROVED** [review]

**Review by tamasd — APPROVED** [review]

**Pull request merged**